packages/loader/driver-utils/src/test/runWithRetry.spec.ts

```diff
@@ -173,18 +187,20 @@ describe("runWithRetry Tests", () => {
 				}),
 			);
 			assert.fail("Should not succeed");
-		} catch (error) {}
+		} catch {
+			// Intentionally empty: error is expected in this test case
+		}
```

Comment:
Like above, this one leaves me scratching my head. "Should not retry if _what_ is disabled?". Then there's an `onRetry` that feels to me like is an error that indicates a problem with the test if it's thrown, but the empty catch might just swallow. Another one not for this PR but if we can find an owner, let's point them to it.